SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -452,7 +452,9 @@ vector<string> CDriver::GetAllCHTMarkersTag() const {
 
   //The CHT markers can be identified as the markers that are customizable with a BC type HEAT_FLUX or ISOTHERMAL.
   for(iMarker=0; iMarker<nBoundariesMarker; iMarker++){
-    if((config_container[ZONE_0]->GetMarker_All_KindBC(iMarker) == HEAT_FLUX || config_container[ZONE_0]->GetMarker_All_KindBC(iMarker) == ISOTHERMAL) && config_container[ZONE_0]->GetMarker_All_PyCustom(iMarker)){
+    if((config_container[ZONE_0]->GetMarker_All_KindBC(iMarker) == HEAT_FLUX ||
+        config_container[ZONE_0]->GetMarker_All_KindBC(iMarker) == ISOTHERMAL ||
+        config_container[ZONE_0]->GetMarker_All_KindBC(iMarker) == HEAT_TRANSFER) && config_container[ZONE_0]->GetMarker_All_PyCustom(iMarker)){
```

Comment:
Time to cut-paste :)

Reply:
Hm, I am unsure whether `HEAT_TRANSFER` is meant to be in that list or if that is unnecessary. The comment above says
```c++
//The CHT markers can be identified as the markers that are customizable with a BC type HEAT_FLUX or ISOTHERMAL.
```
and in CDriver it is
```c++
/*!
   * \brief Get all the heat transfer boundary markers tags.
   * \return List of heat transfer boundary markers tags.
   */
  vector<string> GetAllCHTMarkersTag() const;
```
Personally I would maybe revert the change here alltogether as I am not sure whether sth is specifically coded to workwith/overwrite some HEAT_FLUX or ISOTHERMAL walls. A clarifying insight here would be highly appreciated 💭